packages/pass-style/src/byteArray.js

```diff
@@ -36,7 +60,7 @@ export const ByteArrayHelper = harden({
     (!!check && check(false, X`Immutable ArrayBuffer expected: ${candidate}`)),
 
   assertRestValid: (candidate, _passStyleOfRecur) => {
-    getPrototypeOf(candidate) === ImmutableArrayBufferPrototype ||
+    getPrototypeOf(candidate) === immutableArrayBufferPrototype ||
```

Comment:
I stared and stared to figure out what's different until I noticed the casing. Your casing is better, thanks.